packages/react/src/components/highlight/Highlight.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import { Highlight } from './Highlight';
+
+export default {
+  component: Highlight,
+  title: 'Components/Highlight',
+  parameters: {
+    controls: { expanded: true },
+  },
+};
+
+export const DefaultHighlight = (args) => <Highlight {...args} />;
+DefaultHighlight.args = {
+  text:
+    'You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to.',
+};
+
+export const HighlightSmall = (args) => <Highlight {...args} />;
+HighlightSmall.args = {
+  text:
+    'You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to.',
+  variant: 'small',
+};
+
+export const HighlightLarge = (args) => <Highlight {...args} />;
+HighlightLarge.args = {
+  text:
+    'You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to.',
+  variant: 'l',
+};
+
+export const HighlightCustomAccentColor = (args) => <Highlight {...args} />;
+HighlightCustomAccentColor.args = {
+  text:
+    'You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to.',
+  theme: {
+    '--accent-line-color': '#009246',
+  },
+};
+
+export const HighlightCustomTextColor = (args) => <Highlight {...args} />;
+HighlightCustomTextColor.args = {
+  text:
+    'You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to.',
+  theme: {
+    '--text-color': '#009246',
+  },
+};
+
+export const DefaultQuote = (args) => <Highlight {...args} />;
+DefaultQuote.args = {
+  text: 'Add an interesting quote here',
+  type: 'quote',
+  reference: 'First name Last name. Title.',
+};
```

Comment:
Small improvement idea: create QuoteArgs and HiglightArgs objects with the most basic properties set, then destructure them in these args and add/change when needed 💡 

Example:
`const QuoteArgs = {
  text: 'Add an interesting quote here',
  type: 'quote',
  reference: 'First name Last name. Title.',
};
`

And then for DefaultQuote story's args:
`
DefaultQuote.args = {
  ...QuoteArgs,
};
`